scripts/install_zds.sh

```diff
@@ -290,124 +306,128 @@ if  ! $(_in "-jdk-local" $@) && ( $(_in "+jdk-local" $@) || $(_in "+full" $@) );
-    if [[ $? == 0 ]]; then
-        unzip -q elasticsearch-${ZDS_ELASTIC_VERSION}.zip 
-        rm elasticsearch-${ZDS_ELASTIC_VERSION}.zip
-        mv elasticsearch-${ZDS_ELASTIC_VERSION} elasticsearch
+    if wget_nv "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ZDS_ELASTIC_VERSION}.zip"; then
+        unzip -q "elasticsearch-${ZDS_ELASTIC_VERSION}.zip"
+        rm "elasticsearch-${ZDS_ELASTIC_VERSION}.zip"
+        mv "elasticsearch-${ZDS_ELASTIC_VERSION}" elasticsearch
 
         # add options to reduce memory consumption
-        print_info "#Options added by install_zds.sh" >> "$es_path/config/jvm.options"
-        print_info "-Xms512m" >> "$es_path/config/jvm.options"
-        print_info "-Xmx512m" >> "$es_path/config/jvm.options"
+        {
+            print_info "#Options added by install_zds.sh"
+            print_info "-Xms512m"
+            print_info "-Xmx512m"
+        } >>"$es_path/config/jvm.options"
 
         # symbolic link to elastic start script
-        ln -s "$es_path/bin/elasticsearch" $ZDS_ENV/bin/
+        ln -s "$es_path/bin/elasticsearch" "$ZDS_ENV/bin/"
     else
         print_error "!! Cannot get elasticsearch ${ZDS_ELASTIC_VERSION}"
         exit 1
     fi
-    cd $ZDSSITE_DIR
+    cd "$ZDSSITE_DIR" || exit
 
     zds_fold_end
 fi
 
-
 # local texlive
-if  ! $(_in "-tex-local" $@) && ( $(_in "+tex-local" $@) || $(_in "+full" $@) ); then
+readonly in_minus_tex_local=$(_in "-tex-local" "$@")
+readonly in_plus_tex_local=$(_in "+tex-local" "$@")
+readonly in_plus_full=$(_in "+full" "$@")
+
+if ! $in_minus_tex_local && ($in_plus_tex_local || $in_plus_full); then
     zds_fold_start "texlive" "* [+tex-local] install texlive"
 
     mkdir -p .local
-    cd .local
+    cd .local || exit
     LOCAL=$ZDSSITE_DIR/.local
 
     # clone
     BASE_REPO=$LOCAL/texlive
     REPO=$BASE_REPO/latex-template
 
-    mkdir -p $BASE_REPO
-    cd $BASE_REPO
+    mkdir -p "$BASE_REPO"
+    cd "$BASE_REPO" || exit
 
-    if [ -d $REPO ]; then # remove previous version of the template
-        rm -rf $REPO
+    if [ -d "$REPO" ]; then # remove previous version of the template
+        rm -rf "$REPO"
     fi
 
-    git clone $ZDS_LATEX_REPO
-    if [[ $? == 0 ]]; then
+    if git clone "$ZDS_LATEX_REPO"; then
         # copy scripts
-        cd $BASE_REPO
-        cp $REPO/scripts/texlive.profile $REPO/scripts/packages $REPO/scripts/install_font.sh .
+        cd "$BASE_REPO" || exit
+        cp "$REPO/scripts/texlive.profile" "$REPO/scripts/packages" "$REPO/scripts/install_font.sh" .
 
         # install fonts
         ./install_font.sh
 
         # install texlive
-        sed -i 's@.texlive@texlive@' texlive.profile  # change directory
-        sed -i "s@\$HOME@$LOCAL@" texlive.profile  # change destination
+        sed -i 's@.texlive@texlive@' texlive.profile # change directory
+        sed -i "s@\$HOME@$LOCAL@" texlive.profile    # change destination
```

Comment:
Je ne suis pas vraiment partisan de l'alignement des commentaires. Ce n'est pas dérangeant ici mais je préfère qu'on l'enlève pour ne pas donner envie de le faire ailleurs. :)
```suggestion
        sed -i "s@\$HOME@$LOCAL@" texlive.profile # change destination
```